crates/javy/README.md

```diff
@@ -33,14 +43,6 @@ context.with(|cx| {
 });
 ```
 
-Create a `Runtime` and use the reference returned by `context()` to add functions and evaluate source code.
```

Comment:
This is an instance of out-of-date documentation. The crate-level docs have a slightly different shape.